doc/modules/model_evaluation.rst

```diff
@@ -671,6 +684,24 @@ binary classification and multilabel indicator format.
     for an example of :func:`precision_recall_curve` usage to evaluate
     classifier output quality.
 
+
+.. topic:: References:
+
+  * C.D. Manning, P. Raghavan, H. Schütze, `Introduction to Information Retrieval
```

Comment:
Can you use the RST citation syntax and use references in the text, so that someone reading it knows what citation belongs to what statement? wiki and IR book should go to the definition. Then you can say "some areas such as computer vision use the interpolated area under the precision-recall curve instead [cite pascal VOC]. This can lead to an overly optimistic measure of classifier performance and has been criticized in the literature [Davis, Flach]. Right now, average_precision_score does not implement any interpolated variant".

Reply:
@amueller Yes, I can use RST citation syntax.  I haven't used it before, but [the docs](http://docutils.sourceforge.net/docs/user/rst/quickref.html#citations) make it look like I only need to add tags like `[Everingham2010]`.

I can also modify the text as suggested.  However, my understanding is that the Pascal VOC paper and Davis / Flach papers discuss two different concepts.  The VOC paper presents the 11-point interpolated average precision.  The Davis / Flach papers do not discuss that but instead criticize linear interpolation of a PR curve.

Reply:
VOC doesn't use 11 point in their code

Reply:
@amueller Commit 0c0f4bf046e501f63a24611f8a18d8fd220d62ad includes all of the changes you recommended during your last review.  I added citation links in `model_evaluation.rst` but used different phrasing.  My reading is that [Manning2008] and [Everingham2010] discuss interpolated AP whereas [Davis2006] and [Flach2015] criticize linear interpolation for AUC of precision-recall curves.

I'll confirm the citation links work once the docs finish building.